docs/hoon/twig/wut-test/hep-case.md

```diff
@@ -11,13 +11,23 @@ union, with no default.
 ## Expands to
 
 ```
+:loop
 :ifno  q
   :lost 
 :if  :fits(p.i.q p)
   q.i.q
 :moar(q t.q)
 ```
 
+```
+|-
+?.  q
+  :lost
+?:  ?=(p.i.q p)
+  q.i.q
+$(q t.q)
+```
+
 ## Syntax
```

Comment:
It is a :lost though: it's only a problem if you try to compile it with a
non-void type, and ++open doesn't discriminate on what the input type is.

On Thursday, 7 July 2016, Joe Bryan <notifications@github.com> wrote:

> In docs/hoon/twig/wut-test/hep-case.md
> <https://github.com/urbit/docs/pull/55#discussion_r70024501>:
>
> >  :ifno  q
> >    :lost
> >  :if  :fits(p.i.q p)
> >    q.i.q
> >  :moar(q t.q)
> >  ```
> >
> > +```
> > +|-
> > +?.  q
> > +  :lost
> > +?:  ?=(p.i.q p)
> > +  q.i.q
> > +$(q t.q)
> > +```
> > +
> >  ## Syntax
>
> Github's not showing it here, but I replaced :lost with ~|(%mint-lost ...)
> ...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/urbit/docs/pull/55/files/8c309518d1325589ec5ca032103f436687191997#r70024501>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABxXhq1SCjnMCUY5qEIrlaMJU4nQICctks5qTdRjgaJpZM4I9D6c>
> .
>